packages/nodes-base/nodes/Airtop/actions/interaction/fill.operation.ts

```diff
@@ -0,0 +1,105 @@
+import {
+	type IExecuteFunctions,
+	type INodeExecutionData,
+	type INodeProperties,
+	NodeApiError,
+} from 'n8n-workflow';
+
+import {
+	validateRequiredStringField,
+	validateSessionAndWindowId,
+	validateAirtopApiResponse,
+} from '../../GenericFunctions';
+import { apiRequest } from '../../transport';
+import { IAirtopResponse } from '../../transport/types';
+import { ERROR_MESSAGES } from '../../constants';
+
+export const description: INodeProperties[] = [
+	{
+		displayName: 'Form Data',
+		name: 'formData',
+		type: 'string',
+		typeOptions: {
+			rows: 4,
+		},
+		required: true,
+		default: '',
+		displayOptions: {
+			show: {
+				resource: ['interaction'],
+				operation: ['fill'],
+			},
+		},
+		description: 'The information to fill into the form written in natural language',
+		hint: 'e.g. "Name: John Doe, Email: john.doe@example.com, Phone: +1234567890"',
+	},
+	{
```

Comment:
I don't think we need to make the timeout configurable since it might not be quick. (see my other comment too)

@danielsht86, do you have a preference on us exposing this?